packages/monaco/src/browser/monaco-diff-editor.ts

```diff
@@ -35,6 +35,7 @@ export class MonacoDiffEditor extends MonacoEditor {
         override?: IEditorOverrideServices,
     ) {
         super(new URI(''), modifiedModel, node, m2p, p2m, options, override);
+        this.documents.add(originalModel);
```

Comment:
@akosyakov why do we do the book-keeping here?

Reply:
look at `MonacoEditor.findByDocument`